sdks/python/apache_beam/examples/complete/juliaset/requirements.txt

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################    
+numpy
```

Comment:
This example already requires numpy in https://github.com/apache/beam/blob/16934836498bfe03cdcfb5f8b77f57c1a9bff795/sdks/python/apache_beam/examples/complete/juliaset/setup.py#L111

Given that you are modifying the example, users may be confused why numpy needs to be specified twice: in a requirements.txt file and in setup.py.

Let's try to use a different pipeline for this test. You could create an example where a requirements file is used, or just a test pipeline that does something trivial, like checking whether a particular library is installed and doesn't do any data processing.